dotnet/src/Agents/Core/Chat/SelectionStrategy.cs

```diff
@@ -0,0 +1,30 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+using System.Collections.Generic;
+using System.Threading;
+using System.Threading.Tasks;
+
+namespace Microsoft.SemanticKernel.Agents.Chat;
+
+/// <summary>
+/// Base strategy class for defining completion criteria for a <see cref="AgentChat"/>.
```

Comment:
This comment is confusing to me, I think this class represents the strategy to select the Agent, is that correct?

Reply:
Yes, copy/paste bug it seems.